home/tests/test_models.py

```diff
@@ -155,15 +155,13 @@ def create_message_value(
         media=None,
         message="",
         variation_messages=None,
-        next_prompt="",
```

Comment:
Do we need tests for the new `buttons` field in here?

Reply:
I've added tests for the validations on the individual button types. I'm unsure how to add tests here for compound values, adding to the dictionary doesn't work. The only test I would want to add here is to ensure you can't add more than 3 buttons

Reply:
Okay I've figured out the compound values, added a test for max 3 buttons.